### PR TITLE
[16.0] [MIG] account_invoice_tax_required

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**
https://github.com/OCA/account-invoicing/issues/1250

### CODE REVIEW

File: account_invoice_tax_required/tests/test_account_move_tax_required.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        )
+        self.partner = self.env.ref("base.res_partner_3")
+        account_user_type = self.env.ref("account.data_account_type_receivable")
+
+        self.account_account = self.env["account.account"]
+        self.account_rec1_id = self.account_account.create(
+            dict(
+                code="cust_acc",
+                name="customer account",
+                user_type_id=account_user_type.id,
```
Comment (adrienpeiffer) [inline account_invoice_tax_required/tests/test_account_move_tax_required.py]:
```suggestion
                account_type="asset_receivable",
```

File: account_invoice_tax_required/tests/test_account_move_tax_required.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            (
+                0,
+                0,
+                {
+                    "product_id": self.product.id,
+                    "quantity": 10.0,
+                    "account_id": self.account_account.search(
+                        [
+                            (
+                                "user_type_id",
```
Comment (adrienpeiffer) [inline account_invoice_tax_required/tests/test_account_move_tax_required.py]:
```suggestion
                                "account_type",
```

File: account_invoice_tax_required/tests/test_account_move_tax_required.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                                "=",
+                                self.env.ref("account.data_account_type_revenue").id,
```
Comment (adrienpeiffer) [inline account_invoice_tax_required/tests/test_account_move_tax_required.py]:
```suggestion
                               "income",
```

**Review by adrienpeiffer — CHANGES_REQUESTED** [review]

File: account_invoice_tax_required/tests/test_account_move_tax_required.py (hunk 2 above)
Comment (ThomasBinsfeld) [reply account_invoice_tax_required/tests/test_account_move_tax_required.py]: Fixed

File: account_invoice_tax_required/tests/test_account_move_tax_required.py (hunk 3 above)
Comment (ThomasBinsfeld) [reply account_invoice_tax_required/tests/test_account_move_tax_required.py]: Fixed

**Review by adrienpeiffer — APPROVED** [review]
Test and Code review

**Review by sbejaoui — APPROVED** [review]

**Review by baimont — APPROVED** [review]

**Review by AnizR — APPROVED** [review]
code LGTM

**Comment by kittiu** [comment]
> /ocabot migration account_invoice_tax_required

@adrienpeiffer not related to this PR, I just miss this ocabot migration feature, can you guide where I can find info about this? Thanks! :)

**Review by apineux — APPROVED** [review]

**Comment by adrienpeiffer** [comment]
@kittiu https://github.com/OCA/oca-github-bot#readme

**Pull request merged**